src/SprykerEco/Zed/Computop/Business/Order/Mapper/PostPlace/SofortMapper.php

```diff
@@ -28,7 +26,7 @@ public function getMethodName(): string
      *
      * @return \Generated\Shared\Transfer\ComputopSofortPaymentTransfer
      */
-    public function getComputopTransfer(PaymentTransfer $paymentTransfer): ComputopSofortPaymentTransfer
+    public function getComputopTransfer(PaymentTransfer $paymentTransfer)
```

Comment:
- We have a lot of the method names like this `getComputopTransfer` in Mapper class
- But I'm not sure that it's correct regarding Convention for Mapper method names `map[<SourceEntityName>[To<TargetEntityName>]]($sourceEntity, $targetEntity)` and functionality - Mappers can be used for transforming / append provided data into an other structure.
- I think we either have to describe the transformation from one structure to another, or we don't need a mapper if we can easily extract the substructure from the main one.

Reply:
Seems this code was created before RFC appeared. My changes have been made for existed classes only to pass stan and php-cs validations.
The whole module need a refactoring to follow all current RFCs